src/core/functions-versions.ts

```diff
@@ -0,0 +1,69 @@
+import { DEFAULT_DOTNET_ISOLATED_VERSION, DEFAULT_DOTNET_VERSION, DEFAULT_NODE_VERSION, DEFAULT_PYTHON_VERSION } from "./constants";
+import { logger } from "./utils/logger";
+
+export function getDefaultVersion(apiLanguage: string): string {
+  let apiVersion;
+  // apiLanguage = apiLanguage.toLowerCase();
+  switch (apiLanguage) {
+    case "node":
+      apiVersion = DEFAULT_NODE_VERSION;
+      break;
+    case "python":
+      apiVersion = DEFAULT_PYTHON_VERSION;
+      break;
+    case "dotnet":
+      apiVersion = DEFAULT_DOTNET_VERSION;
+      break;
+    case "dotnetisolated":
+      apiVersion = DEFAULT_DOTNET_ISOLATED_VERSION;
+      break;
+    default:
+      apiVersion = DEFAULT_NODE_VERSION;
+      break;
+  }
+  return apiVersion;
+}
+
+export function getChoicesForApiLanguage(apiLanguage: string) {
+  // Refer to this for functions and versions - https://learn.microsoft.com/azure/static-web-apps/configuration#selecting-the-api-language-runtime-version
+  logger.silly(`ApiLang: ${apiLanguage}`);
+  let choices = [];
+  switch (apiLanguage) {
+    case "node":
+      choices = [
+        { title: "18", value: "18" },
```

Comment:
is there a better way to generat these choices... for example a method whic can return this array.. we just pass the version number ?

Reply:
added a method for this